app/events/telegram.go

```diff
@@ -286,6 +291,29 @@ func (l *TelegramListener) Submit(ctx context.Context, text string, pin bool) er
 	return nil
 }
 
+// SubmitHTML message to telegram's group with HTML mode
+func (l *TelegramListener) SubmitHTML(ctx context.Context, text string, pin bool) error {
```

Comment:
this selective use of ctx is surprising. Nothing wrong with this ctx by itself, but all the other parts don't seem to care about the context, so the reader may ask "what is so special about this func?"

Reply:
`SubmitHTML` is a copy of `Submit` functions with list of arguments. I don't know why do I need `ctx` here